src/VirtualClient/VirtualClient.Dependencies/NetworkConfigurationSetup.cs

```diff
@@ -301,6 +301,7 @@ private async Task ApplyRcLocalFileCommandsAsync(CancellationToken cancellationT
 
             if (this.DisableFirewall)
             {
+                rcLocalContents.Add("if ! command -v iptables &> /dev/null; then sudo dnf install -y iptables; fi");
```

Comment:
This would restrict us to dnf, do it in the PERF-NETWORK.json profile, there is a LinuxPackageInstallation dependency and add it for the dnf section. Apply it to all the networking workload profiles